cedar-policy/Cargo.toml

```diff
@@ -37,6 +37,10 @@ corpus-timing = []
 
 integration_testing = []
 
+# Experimental features
+experimental = ["partial-eval"]
```

Comment:
Is the intent that `experimental` would turn on all experimental features?  If so can we have a comment documenting that?